src/toil/wdl/wdltoil.py

```diff
@@ -705,9 +705,16 @@ def import_file_from_uri(uri: str) -> str:
             tried.append(candidate_uri)
             try:
                 imported = toil.import_file(candidate_uri)
+            except UnimplementedURLException as e:
+                # We can't find anything that can even support this URL scheme.
+                # Report to the user, they are probably missing an extra.
+                logger.critical('Error: ' + str(e))
+                sys.exit(1)
```

Comment:
I'm doing it this way to avoid a stack trace, because I feel like errors that are really the user's fault should probably just be error messages and not stack traces.

We could raise here, but then I'd need code somewhere else to catch and print the same error and do the same exit.